File: FindGuitarTester.java
public class FindGuitarTester {
    public static void main(String[] args) {
        // Set up Rick's guitar inventory
        Inventory inventory = new Inventory();
        initializeInventory(inventory);

        // Erin comes in and tells Rick what she likes
        Guitar whatErinLikes = new Guitar("", 0, "fender", "Stratocaster", "electric", "Alder", "Alder");
        Guitar guitar = inventory.search(whatErinLikes);

        if (guitar != null) {
            System.out.println("Erin, you might like this " + guitar.getBuilder() + " " + guitar.getModel() +
                    " guitar, made of " + guitar.getBackWood() + " back and sides, " + guitar.getTopWood() + " top. \n" +
                    "You can have it for only $" + guitar.getPrice() + "!");
        } else {
            System.out.println("Sorry, Erin, we have nothing for you.");
        }
    }

    private static void initializeInventory(Inventory inventory) {
        // Add guitars to the inventory....
    }
}

File: FindGuitarTester.cs
public class FindGuitarTester
{
    public static void Main(string[] args)
    {
        // Set up Rick's guitar inventory
        Inventory inventory = new Inventory();
        InitializeInventory(inventory);

        // Erin comes in and tells Rick what she likes
        Guitar whatErinLikes = new Guitar("", 0, "fender", "Stratocaster", "electric", "Alder", "Alder");
        Guitar guitar = inventory.Search(whatErinLikes);

        if (guitar != null)
        {
            Console.WriteLine($"Erin, you might like this {guitar.Builder} {guitar.Model} " +
                              $"guitar, made of {guitar.BackWood} back and sides, {guitar.TopWood} top. \n" +
                              $"You can have it for only ${guitar.Price}!");
        }
        else
        {
            Console.WriteLine("Sorry, Erin, we have nothing for you.");
        }
    }

    private static void InitializeInventory(Inventory inventory)
    {
        // Add guitars to the inventory....
    }
}
